Sources/Common/CustomerIOInstance.swift

```diff
@@ -20,6 +20,10 @@ public protocol CustomerIOInstance: AutoMockable {
         body: RequestBody
     )
 
+    // sourcery:Name=identifyAnonymousEncodable
+    // sourcery:DuplicateMethod=identify
+    func identify(body: Codable)
```

Comment:
Used `Codable` directly because I was getting error with generics

```
Cannot find type 'RequestBody' in scope
```

while the implementation was

```
// sourcery:Name=identifyAnonymousEncodable
// sourcery:DuplicateMethod=identify
func identify<RequestBody: Codable>(
    // sourcery:Type=AnyEncodable
    // sourcery:TypeCast="AnyEncodable(body)"
    body: RequestBody
)
```

Reply:
This might work for customers, but it also might not. I can't confirm unless we had it working on a sample app and so I saw some example code for reference. 

We can address this at a later time once we implement a `identify` call in the sample apps.